src/ui/server/controllers/insurance/business/credit-control/index.ts

```diff
@@ -55,6 +55,7 @@ export const get = (req: Request, res: Response) => {
       FIELD_HINT: FIELDS[FIELD_ID].HINT,
       userName: getUserNameFromSession(req.session.user),
       application: application.business,
+      applicationAnswer: application.business[FIELD_ID],
```

Comment:
Seems that if `applicationAnswer` is passed, we do not also need `application`. Can't see it being used in "single radio"?